package.json

```diff
@@ -18,6 +18,6 @@
     "react-native": ">=0.36.0"
   },
   "dependencies": {
-    "react-native-background-geolocation": "^2.14.2"
+    
```

Comment:
did you add your new geolocation library into this package?

Reply:
@hisavali no, this library itself is the package we're adding as dependency to main Hive app. It used to require it's own dependency - "react-native-background-geolocation", but we don't need it anymore as there is our own implementation